src/pages/academy/components/video-banner/_DBanner.js

```diff
@@ -164,6 +176,9 @@ const Dbanner = ({ featured_video_list, non_featured_video_list }) => {
                         </Flex>
                     </Flex>
                     <VideoCarousel carousel_items={non_featured_video_list} />
+                    <AllVideosButton white="true" to="/academy/videos/">
+                        See all videos
+                    </AllVideosButton>
```

Comment:
I don't think the `white` type is the one you should be using as the hover state is different from the `all articles` button. It should be exactly the same as that one.

Reply:
> I don't think the `white` type is the one you should be using as the hover state is different from the `all articles` button. It should be exactly the same as that one.

I can add a new button in `button.js` and name it `light` for example. make it exactly as same as the button we have right now, but copy the hover of the button used for `all articles`. Also that hover is a bit dark, is it OK to make it like this:

Reply:
Is it different from our current tertiary button? If not, we can use that. Otherwise, we can create a new type. But do check with the designer first on inconsistency if it is a new variant or a slight mistake.